### PR TITLE
Add raml file

### PR DESCRIPTION
**Pull request by atd**

### CODE REVIEW

**Review by nogates — CHANGES_REQUESTED** [review]
I am missing the two updates endpoints, one for updating turns and one for updating groups

File: api.raml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+              },
+              {
+                "order": 2,
+                "groupIds": [ 10 ]
+              }
+            ],
+            "email": "space@lists.mat.com",
+            "notification_subject": "@responsible_users have to clean the space",
+            "notification_body": "This is the monthly reminder for @responsible_users to clean the space"
+          }
```
Comment (nogates) [inline api.raml:89]: what about using JSON api format http://jsonapi.org/ / https://github.com/rails-api/active_model_serializers

Comment (atd) [reply api.raml:89]: Done! Shall I include them in the request body also?

Comment (atd) [reply api.raml:89]: I seems so http://jsonapi.org/format/#crud-creating

Comment (atd) [reply api.raml:89]: I still need to define `type` and `attributes`

Comment (nogates) [reply api.raml:89]: Yep, the type is also mandatory

**Comment by atd** [comment]
Please take a look at it now!

I rebased it on top of current master

**Review by nogates — APPROVED** [review]
awesome!

**Pull request merged**